docs/1-development/quickstart.md

```
@@ -0,0 +1,172 @@
+
+# 離開容器
+exit
+```
+
+### 5️⃣ 驗證服務
+
+開啟瀏覽器，存取以下網址：
+
+| 服務 | URL | 說明 |
+|------|-----|------|
+| 🎨 **前端** | http://localhost:3000 | React 應用程式 |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)